api-report/purchases-js.api.json

```diff
@@ -3210,7 +3302,7 @@
             {
               "kind": "PropertySignature",
               "canonicalReference": "@revenuecat/purchases-js!Product#defaultSubscriptionOption:member",
-              "docComment": "/**\n * The default subscription option for this product. Null if no subscription options are available like in the case of consumables and non-consumables.\n */\n",
+              "docComment": "/**\n * The default subscription option for this product. Null in the case of no subscriptions.\n */\n",

```

Comment:
I think the description was clearer before? Another question, should we mark it as deprecated or will we do that later?

Reply:
> I think the description was clearer before?

Hmm yeah I will revert this.

> Another question, should we mark it as deprecated or will we do that later?

My idea was to not deprecate it. This would become a convenience method to avoid the dev having to cast down the `PurchaseOption` themselves if they need to access the `SubscriptionOption` data (like to get all pricing phases for trials/intro offers).

Reply:
Got it